plugins/net_plugin/net_plugin.cpp

```diff
@@ -3714,7 +3727,7 @@ namespace eosio {
                      ("cid", cid)("n", ptr->block_num())("id", id.str().substr(8,16)));
          }
          if( exception || !obh) {
-            if (!obh) {
+            if (!exception && !obh) {
                if (prev_is_proper_svnn_block || !ptr->is_proper_svnn_block()) {
                   fc_dlog(logger, "unlinkable_block ${bn} : ${id}, previous ${pn} : ${pid}",
                           ("bn", ptr->block_num())("id", id)("pn", block_header::num_from_id(ptr->previous))("pid", ptr->previous));
```

Comment:
This comment is for the `assert(obh);` on line 3743 (I could not put a comment there as it is not changed by the PR). That assert is not necessarily true is `exception` is true.

Reply:
If `exception` is `true` it will return in the `if( exception ` statement.

Reply:
I missed the `return`.